etl/firefox_legacy.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+
+import requests
+
+from .search import create_metrics_search_js
+
+OUTPUT_DIRECTORY = os.path.join("public", "data")
+FUNCTIONS_DIRECTORY = ".netlify"
+PROBES_URL = "https://probeinfo.telemetry.mozilla.org/firefox/all/main/all_probes"
+
+# a bit of a hack, we would ideally make this public data. this data is already implicitly
+# public via the probe search service, so putting this here. probably best to make this
+# a public data set produced by bigquery-etl in the long run though
+PROBE_RECORDED_IN_PROCESSES_URL = "https://sql.telemetry.mozilla.org/api/queries/83152/results.json?api_key=Z1WhC2sRmLXmSwZyUwhIOFQQQAUK3JvGNLZgtZGW"  # noqa
+
+
+def get_legacy_firefox_metric_summary():
+    # pull down the recorded in process information, which we use as the
```

Comment:
out of curiosity, is there a reason for when you use docstrings for function description ([example](https://github.com/mozilla/glean-dictionary/blob/ce84fc6b33bc58127792db0b74d46d9217bb03c6/etl/__main__.py?_pjax=%23js-repo-pjax-container%3Afirst-of-type%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%3Afirst-of-type%2C%20%5Bdata-pjax-container%5D%3Afirst-of-type#L76)) and when you don't? I'm trying to learn what's the good practice for Python function documentation.

Reply:
Unfortunately I don't have a great answer here. Ideally we would have docstrings for every function.

https://github.com/mozilla/lookml-generator is a much better example to work from than the Glean Dictionary (I believe @relud even hooked up CI to make sure that every public function has a docstring).

I'll update at least this one to have a docstring.